Backend/Underdogs-Struggling/src/main/java/com/TheTroisMousquetaires/Underdogs/Struggling/service/impl/EntryServiceImpl.java

```diff
@@ -46,4 +45,14 @@ public void deleteEntry(long id){
         entryRepository.deleteEntryByEntryID(id);
     }
 
+    @Override
+    public List<Entry> getAllEntries() {
+        return List.of();
+    }
+
+    @Override
+    public List<Entry> findEntriesByTopic(String topic) {
+        return List.of();
+    }
+
```

Comment:
Why do we have 2 empty functions here? Please comment with a TO DO sign if they are intentionally left blank.